### PR TITLE
Enemy spawning & object

### PR DESCRIPTION
**Pull request by Shom770**

### CODE REVIEW

File: src/resources/enemy.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        self.current_level = current_level
+
+    def draw(self) -> None:
+        """Draws enemies to board"""
+        # while loop to ensure that enemies don't get placed in the same area
+        while str(self.current_level.board[self.y][self.x]) in (self.symbol, '$'):
+            self.x = randint(2, self.current_level.width - 2)
+            self.y = randint(2, self.current_level.width - 2)
+
+        self.current_level.board[self.y][self.x] = self.symbol
```
Comment (n0remac) [inline src/resources/enemy.py:19]: Draw should only place a single enemy on the board where ever it's x and y position is.

**Review by n0remac — CHANGES_REQUESTED** [review]

File: src/resources/enemy.py (hunk 1 above)
Comment (Shom770) [reply src/resources/enemy.py:19]: Weren't we having multiple enemies on the board? with a cap size of 10

File: app.py [hunk 2]
```diff
@@ -15,14 +17,22 @@ def run_game(layout: Layout) -> Panel:
     Layout: Layout  Holds all the rich renderables for the game. Updated with a new panel each tick.
     """
     player.draw()
+    for x in current_node.files:
+        globals()[x.name[:-3]].draw()
+
```
Comment (n0remac) [inline app.py:22]: I have no idea whats going on here.

**Review by n0remac — CHANGES_REQUESTED** [review]

File: src/resources/enemy.py (hunk 1 above)
Comment (n0remac) [reply src/resources/enemy.py:19]: Yes, but that is not what draw does. An enemies draw only draws a single enemy. Every enemy entity will have its own draw.

Comment (n0remac) [reply src/resources/enemy.py:19]: for now start with a single enemy so you can get its movement working more easilly.

Comment (Shom770) [reply src/resources/enemy.py:19]: Ah alright

Comment (Shom770) [reply src/resources/enemy.py:19]: Draw already only places a single enemy on the board though, I just have a for loop in the ``app.py`` to place all the enemies

**Review by n0remac — APPROVED** [review]

**Pull request merged**